README.md

```diff
@@ -64,19 +65,30 @@ To test the bot, you can:
 - `name` is a human-readable name
 - `weight` is a number that will be used to calculate the bounty price
 
+`command-settings` are setting to enable or disable a command
+
+- `name` is the name of the command
+- `enabled` is a `true` or `false` value to enable or disable a command
+
 `default-labels` are labels that are applied when an issue is created without any time or priority labels.
 
-`auto-pay-mode` can be `true` or `false` that enables or disables automatic payout of bounties when the issue is closed.
+`assistive-pricing` to create a new pricing label if it doesn't exist. Can be `true` or `false`.
+
+`disable-analytics` can be `true` or `false` that disables or enables weekly analytics collection by Ubiquity.
 
-`analytics-mode` can be `true` or `false` that enables or disables weekly analytics collection by Ubiquity.
+`payment-permit-max-price` sets the amount for automatic payout of bounties when the issue is closed
 
-`incentive-mode` can be `true` or `false` that enables or disables comment incentives. These are comments in the issue by either the creator of the bounty or other users.
+`comment-incentives` can be `true` or `false` that enable or disable comment incentives. These are comments in the issue by either the creator of the bounty or other users.
 
 `issue-creator-multiplier` is a number that defines a base multiplier for calculating incentive reward for the creator of the issue.
 
 `comment-element-pricing` defines how much is a part of the comment worth. For example `text: 0.1` means that any text in the comment will be multiplied by 0.1
```

Comment:
Will add 0.1

Not sure why my GitHub client isn't letting me suggest changes anymore